src/index.ts

```diff
@@ -16,4 +16,10 @@
 
 export * from './plugin';
 export * from './api';
-export { Router, isPluginApplicableToEntity } from './components/Router';
+export {
+  Router,
+  isPluginApplicableToEntity,
+  isMoreThanOneFirebaseFunction,
```

Comment:
I'm on the fence about wether we should expose this to the application. Are we intending these to be used in the EntityPage?